docs/source/entities.mdx

```diff
@@ -53,10 +53,12 @@ type Product @key(fields: "id") {
 }
 ```
 
-The `@key` directive defines the entity's **primary key**, which consists of one or more of the type's `fields`. In the example above, the `Product` entity's primary key is its `id` field. **Every instance of an entity must be uniquely identifiable by its `@key` fields.** This is what enables your router to associate field data from _different_ subgraphs with the _same_ entity instance.
+The `@key` directive defines the entity's **unique key**, which consists of one or more of the type's `fields`. In the example above, the `Product` entity's unique key is its `id` field. **Every instance of an entity must be uniquely identifiable by its `@key` fields.** This is what enables your router to associate field data from _different_ subgraphs with the _same_ entity instance.
 
 In most cases, the `@key` field(s) for the same entity will be the same across subgraphs. For example, if one subgraph uses `id` as the `@key` field for the `Product` entity, other subgraphs should do the same. However, this [isn't strictly required](./entities-advanced#differing-keys-across-subgraphs).
 
+> If coming from a database context, it can be helpful to think of a `@key` as an entity's [**primary key**](https://en.wikipedia.org/wiki/Primary_key). The term isn't completely accurate for entities since a single entity can have [multiple `@key`s](./entities-advanced#multiple-keys). The field(s) you select for an entity's `@key` must, however, uniquely identify the entity.
```

Comment:
I am comfortable with making this association since most people have at least a good idea around PKs. However, I will also suggest an alternative "candidate key" which is a more accurate fit for the concept.
https://en.wikipedia.org/wiki/Candidate_key

Reply:
I'm not suggesting that we change all the unique key terminology to candidate key though. Candidate key is probably a _bit_ too specific to make it the broad-use term.

Reply:
@trevor-scheer - is an entity having multiple `@keys` analogous to a database having composite keys? If so, is that worth describing here?

Reply:
@shorgi not quite. `@key`s can _also_ be composite. As long as each `@key` (on its own) can uniquely identify an entity within its respective subgraph, it's valid. As an example, maybe a product can be uniquely identified by its sku OR the combination of model + variant + color + size. That would look like:

```graphql
type Product @key(fields: "id") @key(fields: "model variant color size") {
  sku: String!
  model: String!
  variant: String!
  color: String!
  size: String!
  # ...
}

The main differentiation here is that tables only have one primary key.

Reply:
(@shorgi , just FYI you can create composite/compound `@keys` too: https://www.apollographql.com/docs/federation/entities-advanced/#compound-keys, but as Trevor mentioned, we're differentiate vs having multiple keys (composite or not) here.)

Reply:
```suggestion
> If coming from a database context, it can be helpful to think of a `@key` as an entity's [**primary key**](https://en.wikipedia.org/wiki/Primary_key). This term isn't completely accurate for entities since a single entity can have [multiple `@key`s](./entities-advanced#multiple-keys). The field(s) you select for an entity's `@key` must, however, uniquely identify the entity. In that way, `@key`s are similar to [candidate keys](https://en.wikipedia.org/wiki/Candidate_key).
```